examples/transformers/MLFlow_X_HuggingFace_Finetune_a_text_classification_model.ipynb

```diff
@@ -0,0 +1,5244 @@
+            "210dc0cef395402ab533ba3f2f73ec29",
+            "e5edea097f0a4479aaa234322ba9f2b9",
+            "689d497676884fafa3a918f85f8f5060",
+            "5684f7e47ca94ae2a36bd01a817f1c21",
+            "9033f65547a54023afa4670523588642",
+            "059507fad8fa46a38e95b9e1dae90e35",
+            "aa70fb7c32b9457aaa7d62cea535c562",
+            "db65ce2c3d634438ba54fd8d01e5ae13",
+            "b20b59630cef497689f4b200e53462dc",
+            "a16de79c84e948729f4eadfb78e4a08a",
+            "38f011ca6f4848db9186ccd4b2e06557",
+            "ef11eaf6bfc041c38fed5cdfe17caa7d",
+            "a97d0dc77a3e4e548562ede741b59b8d",
+            "c2662f6c97064bbda25147eb1dfea6cd",
+            "f2772a70004d4fa590e354a5bfcefe38",
+            "b9ee74390de74172bab4f33ba09445e3",
+            "3afd0e1210a84cc2bea11a2de30dc1b5",
+            "6df8edba00a7490f8006f88f32c2c7a0",
+            "0ca8528a89d74560b8ea37803e4ad5a5",
+            "06d716e50db745569f50b6add9f2e31d",
+            "df8298f37600471ea7adfe4cc2cee0fe",
+            "98c2c2bfb47f4cffa88808b3b8a8ad73",
+            "09868dddc3b54996a6f2b79c0e4c9b50",
+            "e3e81446a5e54a6cbdf1d63200d43794",
+            "cfdef1af04024448bc490adbf11dc0e2",
+            "11cbc6948a70403f925374a2649cdbf9",
+            "df5b5bf0776c4198aa295d118534ab9e",
+            "67f26a105b9e444e811b48aabf7a147f",
+            "128994fac80044e2af3f80f599f95980",
+            "36d13631dc464f3f93a707cdaa304b84",
+            "87ca65261da04b919e0b348fc883d431",
+            "467d9c289f9241e59abe3f289b5fa859",
+            "a56d59fa5582461c876ba156fd7a5431",
+            "23b71258709642d79244244b6c5979a6",
+            "8c599bd782264ab9a691d35b07dafbcc",
+            "dbceb5bb3de24e26a448566d579bd2ce",
+            "b89bd378e2924c56a7eb2e14d8a404d0",
+            "0d8591fd63df49e5b4ab87898d83b5f0",
+            "4995e0525198465595d84607e11742fc",
+            "ce3a053e600e4b9d888afb10bde0b90a",
+            "6febc7a30bef46cf8cae24ccafa233a8",
+            "31733bed0c714fb29af6eb4de3015068",
+            "cc65f44444004fc28da78ef4fc9cad9e",
+            "5cf9d8b108834f488f57ce7eaf354da5",
+            "ad06b8241c504d0d80dc27baf3f92dae",
+            "1a8e7e67498043a08510fdc920d085a4",
+            "a8f0abb761c848798293574744066d43",
+            "877fedf3d6d94fe0b00397bab89b730a",
+            "8cc03ebce5a14d46910c4ba1ed027053"
+          ]
+        },
+        "id": "Fd2S6Q_jMbfo",
+        "outputId": "02d4b436-e1cf-49b7-82be-067c6a451a31"
+      },
+      "outputs": [],
+      "source": [
+        "from datasets import load_dataset\n",
+        "\n",
+        "# Load \"sms_spam\" dataset.\n",
+        "sms_dataset = load_dataset(\"sms_spam\")\n",
+        "\n",
+        "# Split train/test by 8:2.\n",
+        "sms_train_test = sms_dataset[\"train\"].train_test_split(test_size=0.2)\n",
+        "train_dataset = sms_train_test[\"train\"]\n",
+        "test_dataset = sms_train_test[\"test\"]"
```

Comment:
If I want to reduce the dataset size, what code do I need add?

Reply:
Can we sample data to run the notebook faster?

Reply:
They can slice the dataset object as a python array as I understand. Changing the number of epochs will be the easiest way. 5min is pretty short comparing to other Tensorflow/Keras tutorials, so I am worrying much about the time cost.